fero/analysis.py

```diff
@@ -256,15 +258,16 @@ def _flatten_result(self, result: dict, prediction_row: dict) -> dict:
         """Flattens nested results returned by the api into a single dict and combines it with the provided data"""
         flat_result = {}
         cols = prediction_row.keys()
+
         for target, values in result["data"].items():
-            low_col = self._make_col_name(f"{target}_low", cols)
-            mid_col = self._make_col_name(f"{target}_mid", cols)
-            high_col = self._make_col_name(f"{target}_high", cols)
+            is_v2 = "low90" in values["value"] and "low50" in values["value"]
```

Comment:
I don't love duck typing in the public facing (and browsable) api.  Is it a huge pain to simply indicate the prediction type from Fero?

Reply:
Yeah I'm not in love with it either. Would be nice to have an explicit version: 2 field on this somewhere